packages/python/plotly/plotly/colors/__init__.py

```diff
@@ -0,0 +1,23 @@
+"""For a list of colors available in `plotly.colors`, please see
+
+* the `tutorial on discrete color sequences <https://plotly.com/python/discrete-color/#color-sequences-in-plotly-express>`_
```

Comment:
let's also add the tutorial on continuous color: https://plotly.com/python/colorscales/

Reply:
ok